boardClass.lua

```diff
@@ -416,6 +415,7 @@ function turnOnNotingMode(bool, board)
     board.boardData.isNoting = true
     if board.boardData.selected ~= nil and board.boardData.selected.status ~= status.Given  then
       board.boardData.selected.number = 0
+      board.boardData.selected.status = status["Empty"] 
```

Comment:
when we initiate notation mode we clean the previous status. 
so that if there was a number provided by user, then status won't be equal to Given or Guessed.
otherwise if we initiate notation mode, `togglePossibleNumber` won't work because it checks if status is Empty.